vtex-apps/login
# Feature/query string redirect

#### What is the purpose of this pull request?

Add a query string parameter to directly redirect a user to an OAuth provider in the start of the login

#### What problem is this solving?

When the Query String Parameter `oAuthRedirect` is present, the user is redirected to the corresponding OAuth provider in the start of the login (more info at the docs)

#### How should this be manually tested?

Check out the login component in the following workspace with and without the query string `oAuthRedirect`. It should work only for valid values (Facebook/Google). You can also test it in the home page, in the header's login (same urls without `/login`)
https://rafaprtest--storecomponents.myvtex.com/login
https://rafaprtest--storecomponents.myvtex.com/login?oAuthRedirect=WHATEVER
https://rafaprtest--storecomponents.myvtex.com/login?oAuthRedirect=Facebook
https://rafaprtest--storecomponents.myvtex.com/login?oAuthRedirect=Google

#### Screenshots or example usage

![image](https://user-images.githubusercontent.com/22064061/66956765-5ddcd100-f03b-11e9-8778-979f375ac683.png)

#### Types of changes
